hr_overtime_report/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+{
+    "name": "HR Overtime Report",
+    "summary": "Overtime report in Employees",
+    "version": "18.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/hr-attendance",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": [],
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "hr",
```

Comment:
issue: wrong dependencies.

IIUC, the `hr.attendance.overtime` is defined in `hr_attendance` instead:


```suggestion
        "hr_attendance",
```